src/tests/stub_libcomm.cc

```diff
@@ -49,9 +49,6 @@ Comm::ConnOpener::ConnOpener(const Comm::ConnectionPointer &, const AsyncCall::P
     void Comm::IoCallback::selectOrQueueWrite() STUB
     void Comm::IoCallback::cancel(const char *) STUB
     void Comm::IoCallback::finish(Comm::Flag, int) STUB
-    Comm::CbEntry *Comm::iocb_table = nullptr;
-void Comm::CallbackTableInit() STUB
-void Comm::CallbackTableDestruct() STUB
```

Comment:
It looks like these (correct) changes trigger reformatting of other/unaffected lines in this file, and CI formatting check [fails](https://github.com/squid-cache/squid/actions/runs/13420264076/job/37490932084?pr=1998). The existing and future formatting look inconsistent/bad in this file, but this PR is not about improving source code formatting, so please just apply whatever formatting changes astyle forces us to use here, increasing the diff but fixing CI checks.

Reply:
done in [f381d67](https://github.com/squid-cache/squid/pull/1998/commits/f381d671c09bb8d6a8e70dea0d6ef4bd49ffef7b)